bin/redis-tls

```diff
@@ -13,8 +13,3 @@ REDIS_URL="$(echo $REDIS_URL | sed "s/:[0-9][0-9]*$/:$REDIS_PORT/g" )"
 # Unset our temporary REDIS_PORT variable since we've now moved it into the
 # REDIS_URL.
 unset REDIS_PORT
-
-# Configure the TLS settings for our Redis connection
-# TODO: Currently it's not possible to actually verify the TLS used by Heroku
-#       Redis, so we'll comment this out for a future enhancement.
-# REDIS_URL="$REDIS_URL?ssl_cert_reqs=required&ssl_ca_certs=$(python -m certifi)"
```

Comment:
I think we can actually uncomment this line now? @ewdurbin would know for sure.

Reply:
Confirmed! Uncomment and remove the TODO!